src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.js

```diff
@@ -13,24 +14,47 @@ const STORAGE_DASHBOARDS_ACCORDION_EXPANDED_KEY = 'dashboardsAccordionExpanded';
 
 const ScenarioDashboardCard = () => {
   const { t } = useTranslation();
-  const { hasScenarioBeenRun, isDashboardSync } = useScenarioDashboardCard();
+  const { hasRunBeenSuccessful, isDashboardSync, downloadCurrentScenarioRunLogs } = useScenarioDashboardCard();
 
   const [isDashboardsAccordionExpanded, setIsDashboardsAccordionExpanded] = useState(
     localStorage.getItem(STORAGE_DASHBOARDS_ACCORDION_EXPANDED_KEY) === 'true'
   );
+
   const toggleDashboardsAccordion = () => {
     const isExpanded = !isDashboardsAccordionExpanded;
     setIsDashboardsAccordionExpanded(isExpanded);
     localStorage.setItem(STORAGE_DASHBOARDS_ACCORDION_EXPANDED_KEY, isExpanded);
   };
 
+  const downloadLogsButton = useMemo(() => {
+    if (!hasRunBeenSuccessful) return null;
+
+    const triggerLogsDownload = (event) => {
+      event.stopPropagation();
+      downloadCurrentScenarioRunLogs();
+    };
+    const color = isDashboardSync ? 'primary' : 'inherit';
```

Comment:
maybe we can define the color inside the component (the condition is rather simple to create a special `const` for) ?

Reply:
Good idea :+1: